internal/kafka/test/mocks/kafkas/kafka_instance_types.go

```diff
@@ -0,0 +1,35 @@
+package mocks
+
+import "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+
+const (
+	DefaultKafkaInstanceSizeId         = "x1"
+	DefaultIngressThroughputPerSec     = "1Mi"
+	DefaultEgressThroughputPerSec      = "1Mi"
+	DefaultTotalMaxConnections         = 100
+	DefaultMaxDataRetentionSize        = "1Gi"
+	DefaultMaxDataRetentionPeriod      = "P14D"
+	DefaultMaxConnectionAttemptsPerSec = 100
+	DefaultQuotaconsumed               = 1
+	DefaultQuotaType                   = "RHOSAK"
+	DefaultCapacityConsumed            = 1
+)
+
+func BuildKafkaInstanceSize(modifyFn func(kafkaInstanceSize *config.KafkaInstanceSize)) *config.KafkaInstanceSize {
```

Comment:
this is a good candidate for the functional options pattern

Reply:
updated it there, some function and var names were clashing with the kafka mock so i put it in its own package. Please take a look when you get a chance :)